src/assets/i18n/de_DE.json

```diff
@@ -2654,5 +2654,7 @@
   "registration.tac_privacy_policy.end.text": ".",
   "registration.tac.error.tip": "Bitte stimmen Sie den Bedingungen zu, um fortzufahren.",
   "registration.privacy_policy.details.title": "Datenschutzhinweise",
-  "customer.credentials.passwordreset.invalid_password.error.PasswordExpressionViolation": "Das Kennwort muss mindestens 7 Zeichen enthalten, bestehend aus Zahlen und Buchstaben, keine Leerzeichen."
+  "customer.credentials.passwordreset.invalid_password.error.PasswordExpressionViolation": "Das Kennwort muss mindestens 7 Zeichen enthalten, bestehend aus Zahlen und Buchstaben, keine Leerzeichen.",
+  "session.timeout.notification.title": "Logged out",
```

Comment:
Should be German strings

Reply:
Yes, to be done by Avengers. Currently I'm waiting for UX to say, what the messages should be.